cernopendata/modules/fixtures/data/docs/news/html/atlas_17022015.html

```diff
@@ -1,4 +1,4 @@
-<p>The dataset from the ATLAS Higgs Machine Learning Challenge has <a href=\"/search?page=1&size=20&q=atlas%20higgs%202014\">been released</a> on CERN Open Data Portal.</p>
-<p><a href=\"http://atlas.cern/updates/atlas-news/are-you-higgs-challenge\">The Challenge</a>, which ran from May to September 2014, was to develop an algorithm that improved the detection of the Higgs boson signal. The <a href=\"https://home.cern/about/updates/2013/11/atlas-sees-higgs-boson-decay-fermions\">specific sample</a> used simulated Higgs particles into two tau particles inside the ATLAS detector. The downloadable sample was provided for participants at the host platform on Kaggle’s website. With almost 1,785 teams competing, the event was a huge success. Participants applied and developed cutting edge Machine Learning techniques, which have been shown to be better than existing traditional high-energy physics tools.</p>
+<p>The dataset from the ATLAS Higgs Machine Learning Challenge has <a href=\search?page=1&size=20&q=atlas%20higgs%202014>been released</a> on CERN Open Data Portal.</p>
```

Comment:
Please fix also the other news links, there are more:

```console
grep -c href ./cernopendata/modules/fixtures/data/docs/news/html/*
./cernopendata/modules/fixtures/data/docs/news/html/alice_20112014.html:0
./cernopendata/modules/fixtures/data/docs/news/html/atlas_17022015.html:3
./cernopendata/modules/fixtures/data/docs/news/html/atlas_29072016.html:0
./cernopendata/modules/fixtures/data/docs/news/html/cms_20112014.html:1
./cernopendata/modules/fixtures/data/docs/news/html/cms_22042016.html:2
```

and make sure that the produced URL is properly quoted `<a href="...">...</a>` .

...or maybe just make sure this is all OK in the docs-to-markdown PR #2231, and therefore this PR can be closed?

Reply:
Yep I think it's better to go with #2231

Reply:
OK, closing the PR...